### PR TITLE
add newline before table 🤞🏻

### LINKED ISSUE #2127: Shell completion table is broken in README
As @jfy133 report, the table doesn't render correctly:  https://nf-co.re/tools/#activate-shell-completions-for-nf-coretools

### PR DESCRIPTION
**Pull request by mashehu**
Fixes #2127 (hopefully)

### CODE REVIEW

**Review by ewels — APPROVED** [review]
The fact that prettier is now formatting the table whitespace for you is a good sign! 👍🏻

File: README.md [hunk 1]
```diff
@@ -217,11 +217,12 @@ Please refer to the respective documentation for further details to manage packa
...
-shell | shell config file | command
---- | --- | ---
-bash | ~/.bashrc | `eval "$(_NF_CORE_COMPLETE=bash_source nf-core)"`
-Zsh | ~/.zshrc | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`
-fish | ~/.config/fish/completions/nf-core.fish | `eval (env _NF_CORE_COMPLETE=fish_source nf-core)`
+
+| shell | shell config file                       | command                                            |
+| ----- | --------------------------------------- | -------------------------------------------------- |
+| bash  | ~/.bashrc                               | `eval "$(_NF_CORE_COMPLETE=bash_source nf-core)"`  |
+| Zsh   | ~/.zshrc                                | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`   |
```
Comment (ewels) [inline README.md:224]: Lower case?

```suggestion
| zsh   | ~/.zshrc                                | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`   |
```

File: README.md [hunk 2]
```diff
@@ -217,11 +217,12 @@ Please refer to the respective documentation for further details to manage packa
...
+| fish  | ~/.config/fish/completions/nf-core.fish | `eval (env _NF_CORE_COMPLETE=fish_source nf-core)` |
```
Comment (ewels) [inline README.md:225]:
```suggestion
| bash  | `~/.bashrc`                               | `eval "$(_NF_CORE_COMPLETE=bash_source nf-core)"`  |
| Zsh   | `~/.zshrc`                                | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`   |
| fish  | `~/.config/fish/completions/nf-core.fish` | `eval (env _NF_CORE_COMPLETE=fish_source nf-core)` |
```

File: README.md (hunk 1 above)
Comment (mashehu) [reply README.md:224]: the authors don't seem to be consistent about it,think I prefer lower case, but I was following how Wikipedia writes it 🤷🏻

**Comment by mashehu** [comment]
@nf-core-bot fix linting

**Review by mirpedrol — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -217,11 +217,12 @@ Please refer to the respective documentation for further details to manage packa
...
-shell | shell config file | command
---- | --- | ---
-bash | ~/.bashrc | `eval "$(_NF_CORE_COMPLETE=bash_source nf-core)"`
-Zsh | ~/.zshrc | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`
-fish | ~/.config/fish/completions/nf-core.fish | `eval (env _NF_CORE_COMPLETE=fish_source nf-core)`
+
+| shell | shell config file                         | command                                            |
+| ----- | ----------------------------------------- | -------------------------------------------------- |
+| bash  | `~/.bashrc`                               | `eval "$(_NF_CORE_COMPLETE=bash_source nf-core)"`  |
+| Zsh   | `~/.zshrc`                                | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`   |
```
Comment (mashehu) [inline README.md:224]:
```suggestion
| zsh   | `~/.zshrc`                                | `eval "$(_NF_CORE_COMPLETE=zsh_source nf-core)"`   |
```

**Pull request merged**